tools/postgres/schema.sql

```diff
@@ -553,7 +553,7 @@ CREATE TYPE webknossos.AI_MODEL_CATEGORY AS ENUM ('em_neurons', 'em_nuclei', 'em
 
 CREATE TABLE webknossos.aiModels(
   _id CHAR(24) PRIMARY KEY,
-  _organization VARCHAR(256) NOT NULL,
+  _owningOrganization VARCHAR(256) NOT NULL,
```

Comment:
It might be more consistent with the other code to leave this `_organization` and rather rename the others to `sharedOrganizations`. That would be like what we have with annotations and teams. What do you think?

Reply:
I see your point, and ultimately it’s not that important. But I still that if the organizations the model is shared with were called webknossos.aiModel_sharedOrganizations (which would be consistent with sharedTeams *and* more descriptive than just organizations), the semantics of `_organization` would be fairly clear. This is the organization the aiModel lives in / belongs to, just like with users, datasets, teams etc..

Reply:
Ok I switched back to `_organization`